### PR TITLE
Add link to Find More Apps button to OzoneConfig

### PR DESCRIPTION
**Pull request by SOAP1914**

### CODE REVIEW

**Comment by rpokorny** [comment]
Is there a reason that a separate MORE_APPS_URL was added instead of just using CENTER_URL?

File: src/app/addApplicationsModal/addApplicationsModal.tpl.html [hunk 1]
```diff
@@ -5,7 +5,9 @@
       <span class="sr-only">Close</span>
     </button>
     <h3 class="modal-title">Open bookmarked apps</h3>
-    <button type="button" class="btn btn-default pull-right">Find more apps</button>
+    <a href="{{moreAppsUrl}}">
```
Comment (arw180) [inline src/app/addApplicationsModal/addApplicationsModal.tpl.html]: this won't work (or at least is not guaranteed to work) - you need to use `ng-href` for links from ng templates: https://docs.angularjs.org/api/ng/directive/ngHref

File: src/OzoneConfig.js [hunk 2]
```diff
@@ -3,5 +3,6 @@ window.OzoneConfig = {
     'CENTER_URL': 'https://ozone-development.github.io/center-ui',
     'HUD_URL': 'https://ozone-development.github.io/hud-ui',
     'WEBTOP_URL': 'https://ozone-development.github.io/ozp-webtop',
-    'METRICS_URL': '/path/to/metrics'
+    'METRICS_URL': '/path/to/metrics',
+    'MORE_APPS_URL': 'https://ozone-development.github.io/center-ui'
```
Comment (arw180) [inline src/OzoneConfig.js]: Just use the existing CENTER_URL link for this

**Comment by arw180** [comment]
Commit message should follow conventional changelog format: https://github.com/ajoslin/conventional-changelog/blob/master/CONVENTIONS.md (message for this commit might be:

fix(addApplicationsModal): Fixes broken link on 'Find more apps' button

Find more apps button now redirects user to Center

Closes #437

**Pull request merged**